### PR TITLE
Document constructor

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/level.js/issues/103

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,20 @@ Not to be confused with [leveljs](https://www.npmjs.com/package/leveljs).
 
 This library is best used with [browserify](http://browserify.org).
 
+## API
+
+### `db = leveljs(location[, options])`
+Returns a new `leveljs` instance. `location` is a String pointing to the IndexedDB location to be opened.
```
Comment (vweevers) [inline README.md]:
```diff
-`location` is a String pointing to the IndexedDB location to be opened
+`location` is the string name of the IndexedDB object store to be opened
```

Comment (vweevers) [reply README.md]: Nah, that's not quite it either. It also controls the IndexedDB database name.

Comment (ralphtheninja) [reply README.md]: @vweevers Think about it for a bit. I'll do other stuff meanwhile.

Comment (vweevers) [reply README.md]: How about: `location` is the string name of the [`IDBDatabase`](https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase) to be opened, as well as the object store within that database. The database name will be prefixed with `options.prefix`.

File: README.md [hunk 2]
```diff
@@ -58,6 +58,20 @@ Not to be confused with [leveljs](https://www.npmjs.com/package/leveljs).
...
+## API
+
+### `db = leveljs(location[, options])`
+Returns a new `leveljs` instance. `location` is the string name of the IndexedDB object store to be opened.
+
+#### `options`
+
+The optional `options` argument may contain:
+
+* `prefix` *(string, default: `'level-js-'`)*: Prepended to the location for the underlying IndexedDB store.
```
Comment (vweevers) [inline README.md]: And here we can say: "Prefix for `IDBDatabase` name"

**Pull request merged**